CefSharp.OutOfProcess.Wpf.HwndHost/ChromiumWebBrowser.cs

```diff
@@ -1070,6 +1092,11 @@ public IChromiumWebBrowser WebBrowser
         public static readonly DependencyProperty WebBrowserProperty =
             DependencyProperty.Register(nameof(WebBrowser), typeof(IChromiumWebBrowser), typeof(ChromiumWebBrowser), new UIPropertyMetadata(defaultValue: null));
 
+        /// <summary>
+        /// The requestContextPreferences used for the RequestContext.
+        /// </summary>
+        private readonly IDictionary<string, object> requestContextPreferences;
```

Comment:
Please move this up the top with all the other `private` fields.

Reply:
done